treeherder/webapp/api/jobs.py

```diff
@@ -327,30 +451,51 @@ def bug_suggestions(self, request, project, pk=None):
         return Response(get_error_summary(job))
 
     @detail_route(methods=['get'])
-    @with_jobs
-    def similar_jobs(self, request, project, jm, pk=None):
+    def similar_jobs(self, request, project, pk=None):
         """
         Get a list of jobs similar to the one selected.
         """
-        job = jm.get_job(pk)
-        if not job:
-            return Response("No job with id: {0}".format(pk), status=HTTP_404_NOT_FOUND)
+        try:
+            repository = Repository.objects.get(name=project)
+        except Repository.DoesNotExist:
+            return Response({
+                "detail": "No project with name {}".format(project)
+            }, status=HTTP_404_NOT_FOUND)
+
+        try:
+            job = Job.objects.get(repository=repository,
+                                  project_specific_id=pk)
+        except ObjectDoesNotExist:
+            return Response("No job with id: {0}".format(pk),
+                            status=HTTP_404_NOT_FOUND)
+
+        filter_params = request.query_params.copy()
 
-        query_params = request.query_params.copy()
-        query_params['job_type_id'] = job[0]['job_type_id']
-        query_params['id__ne'] = job[0]['id']
-        url_query_filter = UrlQueryFilter(query_params)
-        offset = int(url_query_filter.pop("offset", 0))
+        offset = int(filter_params.get("offset", 0))
         # we don't need a big page size on this endoint,
         # let's cap it to 50 elements
-        count = min(int(url_query_filter.pop("count", 10)), 50)
-        return_type = url_query_filter.pop("return_type", "dict").lower()
-        results = jm.get_job_list_sorted(offset, count,
-                                         conditions=url_query_filter.conditions)
-
-        # add result set id for backwards compatibility
-        for result in results:
-            result['result_set_id'] = result['push_id']
+        count = int(filter_params.get("value", 50))
```

Comment:
Hmm, here's another example of ``value`` being used.  Is this still going to support the ``count`` param being passed in the querystring?  I know it's used by people.  Sorry if I'm just ignorant of this feature, but I can't find an example of ``value`` meaning ``count`` in the Yahoo!s / DRF docs.

I see ``limit`` here: http://www.django-rest-framework.org/api-guide/pagination/#limitoffsetpagination